File: 프로그래머스/unrated/181855. 문자열 묶기/문자열 묶기.py
def solution(strArr):
    answer = 0
    list =[]
    for i in strArr:
        list.append(len(i))
    most_common_value = max(set(list), key = list.count)
    for i in strArr: 
        if len(i)==most_common_value:
            answer +=1
    return answer
        
    
    

File: 프로그래머스/0/181855. 문자열 묶기/문자열 묶기.js
function solution(strArr) {
    var answer = {};
   let count=0
       for(let i of strArr){
             if (!answer[i.length]) {
                answer[ i.length] =1
           }else{
                answer[ i.length] +=1
           }}
    let maxKey = Object.keys(answer).reduce((a, b) => answer[a] > answer[b] ? a : b);
    for(let i of strArr){
if (i.length==maxKey){
    count++
}}
    return count;
}